src/components/Navbar/index.js

```diff
@@ -3,15 +3,17 @@ import React, { useState } from 'react'
 import LoggedOutLinks from './LinksGroup/LoggedOutLinks'
 import LoggedInLinks from './LinksGroup/LoggedInLinks'
 
-import './styles.scss'
+import classes from './styles.module.scss'
 
 const Navbar = () => {
   const [loggedIn, setLoggedIn] = useState(false)
 
   return (
-    <nav className="navbar">
-      <img className="logo" src="https://placehold.co/205x34" alt="" role="presentation" />
-      <div className="nav-items">
+    <nav className={classes.navbar}>
+      <div>
+        <span />
+      </div>
```

Comment:
Always take into consideration that adding pure html for layout or styling is kind of a code smell. You can probably work this around using two containers with `justify-content: space-between`. But you can leave it like this for now.